docs/installation.md

```diff
@@ -0,0 +1,49 @@
+---
+hide:
+  - navigation
+---
+
+# Installation
+
+## Stable release
+
+To install Taipy, run this command in your terminal:
+
+``` console
+$ pip install taipy
+```
+
+!!! info "This is the preferred method to install Taipy, as it will always install the most recent stable release."
+
+    If you don't have [pip][] installed, this [Python installation guide][]
+    can guide you through the process.
+
+    This method also applies for installing Taipy in an Anaconda environment.
```

Comment:
I would create 2 different sections. One for pip one for conda.

Reply:
I would also provide an example with conda explaining that pip is installed inside the conda virtual env: somewhere like "/anaconda/envs/my_venv_name/"

```
conda install pip
pip install taipy
```

Reply:
Not sure I understand the part "/anaconda/envs/my_venv_name/", but yes I agree with having 2 different sections and also mentioning installing pip in the anaconda env